[CP][Stable] Fix race condition causing crash when interacting with an animating scrollable (#164708)
flutter/flutter: CHANGELOG.md

### [3.29.2](https://github.com/flutter/flutter/releases/tag/3.29.2)

- [flutter/162198](https://github.com/flutter/flutter/issues/162198) Fixes double-download of canvaskit.wasm

> - [flutter/164392](https://github.com/flutter/flutter/pull/164392) - Fix race condition causing crash when interacting with an animating scrollable.

Reviewer: ```suggestion
- [flutter/164392](https://github.com/flutter/flutter/pull/164392) Fixes a crash that can occur when animating and interacting with a scrollable simultaneously.
```
